lib/helpers/copy-fixture-files.js

```diff
@@ -2,13 +2,11 @@
 
 var path    = require('path');
 var Promise = require('rsvp');
-var copy    = Promise.denodeify(require('cpr'));
+var fs = require('fs-extra');
+var copy = Promise.denodeify(fs.copy);
 
 var rootPath = process.cwd();
 
 module.exports = function copyFixtureFiles(sourceDir) {
-  return copy(path.join(rootPath, 'tests', 'fixtures', sourceDir), '.', {
-    clobber: true,
-    stopOnErr: true
-  });
+  return copy(path.join(rootPath, 'tests', 'fixtures', sourceDir), '.', { clobber: true });
```

Comment:
I seem to recall starting with fs-extra's copy, but switched to cpr specifically for the `stopOnErr` option. It may be unnecessary for our use case though.